providers/server_socket.js

```diff
@@ -50,9 +65,9 @@ ServerSocket.prototype.getInfo = function() {
   var port = nsIServerSocket.port;
   var localAddress = '127.0.0.1';
   var socketType = 'tcp';
-  var info = {socketType: socketType,
-              localAddress: localAddress,
-              localPort: port};
+  var info = { socketType   : socketType,
```

Comment:
Yeah I didn't like it much either, just tried to do a minimal change from what it was. Have made it more normal now.